sql/mob_skills.sql

```diff
@@ -2744,9 +2744,9 @@ INSERT INTO `mob_skills` VALUES (2885,2021,'beautiful_death',1,18.0,2000,1000,4,
 -- INSERT INTO `mob_skills` VALUES (2920,2664,'.',0,7.0,2000,1500,4,0,0,0,0,0,0);
 -- INSERT INTO `mob_skills` VALUES (2921,2665,'.',0,7.0,2000,1500,4,0,0,0,0,0,0);
 INSERT INTO `mob_skills` VALUES (2922,2094,'soulshattering_roar',1,18.0,2000,1000,4,0,0,0,0,0,0);
-INSERT INTO `mob_skills` VALUES (2923,2095,'calcifying_claw',1,18.0,2000,1000,4,0,0,0,0,0,0);
-INSERT INTO `mob_skills` VALUES (2924,2096,'divesting_stampede',1,18.0,2000,1000,4,0,0,0,0,0,0);
-INSERT INTO `mob_skills` VALUES (2925,2097,'bonebreaking_barrage',1,18.0,2000,1000,4,0,0,0,0,0,0);
+INSERT INTO `mob_skills` VALUES (2923,2095,'calcifying_claw',0,18.0,2000,1000,4,0,0,1,0,0,0);
```

Comment:
was going to approve this but got confused by changing these values to 0 - does that mean no target? should it be 4 for target enemy for these two entries? apologies if i am missing something!

Reply:
That column corresponds to mob_skill_aoe:
https://github.com/project-topaz/topaz/blob/release/sql/mob_skills.sql#L33

I can't find the enum that describes it, but 0 is single-target:
https://github.com/project-topaz/topaz/blob/4d09173b57dd5867dd3997436fa3ee66eeea0fbb/src/map/mobskill.cpp#L61

Reply:
ahh just me misreading the sql lines - sorry about that, thought it was mob_valid_target